Classes/Utility/AlertActions.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AlertActions.swift
+//  Freetime
+//
+//  Created by Ivan Magda on 25/09/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import UIKit
+
+struct AlertActions {
+    
+    static func share(_ items: [Any],
+                      activities: [UIActivity]?,
+                      controller: UIViewController,
+                      sender: UIBarButtonItem) -> UIAlertAction {
+        weak var weakController = controller
```

Comment:
Do I need weak reference here?

Reply:
Probably recommended as we don't want to hold a reference to the Issue/Repo view controllers just for the sake of sharing!

Reply:
Ok, thank you!